Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -230,21 +230,19 @@ private List<(EntityUid Entity, CargoPalletComponent Component, TransformCompone
 
     #region Station
 
-    private bool SellPallets(EntityUid gridUid, EntityUid? station, out double amount)
+    private bool SellPallets(EntityUid gridUid, out double amount)
     {
-        station ??= _station.GetOwningStation(gridUid);
+        //station ??= _station.GetOwningStation(gridUid);
```

Comment:
just remove it

Reply:
I have no issue with that, but was there any reason for adding station information there?
It really felt like EntitySoldEvent was made with bounties in mind since it had station if as an arg.

Reply:
dont comment out unused code just remove it